versionberg/src/main/groovy/io/github/rockerhieu/versionberg/Git.groovy

```diff
@@ -26,9 +26,12 @@ package io.github.rockerhieu.versionberg
  * Created by rockerhieu on 9/12/16.
  */
 class Git {
+    static String repositoryPath = ''
```

Comment:
I think we should avoid using static stateful variable. How about `getCommitSha(gitDir)` and `getCommitCount(gitDir)`?